### ISSUE TITLE
 DXA 2.1 .\cms-import.ps1 -importType "framework-only" - not working

### ISSUE THREAD

**Issue by avmgan** [sdl/dxa-content-management#33]
I was looking to import only DXA framework to for my development and noticed that framework-only import not working, seems to look like a bug.

```
PS C:\vagrant\share\SDL-Tridion-Sites9\Sites-9-Quick-Install\dxa\cms> .\cms-import.ps1 -importType "framework-only" -cmsUrl "http://vagrant-2016/" -rootPublication "000 Empty" -masterPublication "100 Audora Master" -modelMapping R2 -Debug
Installed CMS version reported to be 9.0.0.609
Copied 'C:\vagrant\share\SDL-Tridion-Sites9\Sites-9-Quick-Install\dxa\cms\DXA Application Data Definition.xml' to 'C:\Tridion\config\ImportExport\Ap
plicationData'
Folder '/webdav/100%20Audora%20Master/Building%20Blocks/Modules/Core' does not exist which implies this is an initial import. If you are upgrading, 
check whether the -masterPublication and -rootFolder parameters are specified correctly.
Uploading package 'C:\vagrant\share\SDL-Tridion-Sites9\Sites-9-Quick-Install\dxa\cms\framework-only.zip' ...
Importing content ...
Aborted
2019/01/07 12:57:59 <31> [Information] Import process 'a061cc125a094a7f826b133dce53a10c' is started by 'VAGRANT-2016\vagrant' at 1/7/2019 12:57:59 P
M.
2019/01/07 12:57:59 <31> [Information] Building list of items to import and determining required import actions.
2019/01/07 12:57:59 <31> [Information] Unzipping and initializing package for import.
2019/01/07 12:58:00 <31> [Error] Tridion.ContentManager.ImportExport.ImportExportException: Item '/webdav/100%20Master/Building%20Blocks/Modules/Cor
e' cannot be imported because it is absent on import TCM and its content was not exported to the package.
   at Tridion.ContentManager.ImportExport.ToDoListInitializer.<>c__DisplayClass8_0.<SetImportActions>b__2()
   at Tridion.ContentManager.ImportExport.Process.<>c__DisplayClass35_0.<InvokeInErrorHandler>b__0(Boolean isRetry)
   at Tridion.ContentManager.ImportExport.AbortErrorHandler.Invoke[T](Func`2 function, LocalizableMessage message, Boolean retryAvailable, Boolean i
sCritical, UserReply additionalUserReplyOptions)
   at Tridion.ContentManager.ImportExport.Process.InvokeInErrorHandler[T](Func`2 function, LocalizableMessage message, Boolean retryAvailable, Boole
an isCritical)
   at Tridion.ContentManager.ImportExport.ToDoListInitializer.SetImportActions()
   at Tridion.ContentManager.ImportExport.Importer.InitializeImport(String packageName)
   at Tridion.ContentManager.ImportExport.Importer.Import(String packageName)
2019/01/07 12:58:00 <31> [Information] Transactional import failed at 1/7/2019 12:58:00 PM. No changes were committed to SDL Web Content Manager.
An error occured while importing 'C:\vagrant\share\SDL-Tridion-Sites9\Sites-9-Quick-Install\dxa\cms\framework-only.zip'
At C:\vagrant\share\SDL-Tridion-Sites9\Sites-9-Quick-Install\dxa\ImportExport\ContentManagerUtils.ps1:422 char:9
+         throw "An error occured while importing '$packageFullPath'"
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (An error occure...ework-only.zip':String) [], RuntimeException
    + FullyQualifiedErrorId : An error occured while importing 'C:\vagrant\share\SDL-Tridion-Sites9\Sites-9-Quick-Install\dxa\cms\framework-only.z 
   ip'
```

Please look into this issue.

**Comment by rpannekoek** [comment]
Internal Issue ID (for tracking purposes): CRQ-13026

**Comment by rpannekoek** [comment]
In DXA 2.1, this feature only works in upgrade scenarios. The plan is to make it work in new install scenarios too in DXA 2.2.

**Comment by rpannekoek** [comment]
Fixed in DXA 2.2
